charts/stack/requirements.yaml

```
@@ -1,12 +1,12 @@
 dependencies:
   - name: mysql-operator
     repository: https://presslabs.github.io/charts/
-    version: ~0.3.9
+    version: ~0.3.10
     condition: mysql-operator.enabled
 
   - name: wordpress-operator
     repository: https://presslabs.github.io/charts/
-    version: v0.8.4
+    version: v0.9.0
```

I think this should be
```suggestion
    version: 0.9.0
```